LuisValgoi/ui5-webcomponents-react-seed
# Session Timeout

## Description

This PR adds the Session Timeout dialog and engine;
The Information Dialog is included since it is a dependency.

## Testing

1. Open Application;
1. Let the application unattended;
1. Wait for 13 minutes (or change the `Constants.js` file);
1. Check that a Warning Popup displays;
1. Wait for 15 minutes (or change the `Constants.js` file);
1. Check that a Timeout Popup displays.

## Automated Tests

- [x] Select this if your code has automated Tests for it.

## References

- https://github.com/LuisValgoi/ui5-webcomponents-react-seed/issues/17
- https://github.com/LuisValgoi/ui5-webcomponents-react-seed/issues/18
